.github/actions/send-slack-notification/compute-message.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+echo 'result<<EOF'
+
+if [ -n "$BLOCK_MESSAGE" ]; then
+    echo -n "${BLOCK_MESSAGE}" | sed -z 's/\n/\\n/g'
+else
+    case $EVENT_NAME in
+        pull_request)
+        echo -n "${PR_TITLE}" | sed -z 's/\n/\\n/g'
+        ;;
+        issues)
+        echo -n "${ISSUE_BODY}" | sed -z 's/\n/\\n/g'
+        ;;
+        *)
+        echo -n "${COMMIT_MESSAGE}" | sed -z 's/\n/\\n/g'
+        ;;
+    esac
+fi
+
+echo ''
```

Comment:
what about removing the `-n` switch from `echo` commands  and removing that empty `echo` ?

Reply:
it is to avoid having that last newline converted to `\n` by sed, while I really want a NL before the closing heredoc tag

`echo "\nEOF"` would have worked as well but for some reason looked more cleaner :D